### PR TITLE
Add support for the conversion endpoint

### LINKED ISSUE #43: Implement `submitConversion`

### PR DESCRIPTION
**Pull request by mheap**
Resolves #43

### CODE REVIEW

File: src/Conversion/Client.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        if ($timestamp) {
+            $params['timestamp'] = $timestamp;
+        }
+
+        $response = $this->client->postUrlEncoded(
+            \Nexmo\Client::BASE_API . '/conversions/'.$type.'?'.http_build_query($params),
+            []
+        );
+
+        if($response->getStatusCode() != '200'){
```
Comment (mheap) [inline src/Conversion/Client.php]: We only check the HTTP code for this API

File: src/Conversion/Client.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            throw $this->getException($response);
+        }
+    }
+
+    protected function getException(ResponseInterface $response)
+    {
+        $body = json_decode($response->getBody()->getContents(), true);
+        $status = $response->getStatusCode();
+
+        if($status === 402) {
```
Comment (mheap) [inline src/Conversion/Client.php]: 402 is special - it means conversions aren't activated on your account

File: src/Conversion/Client.php [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            $e = new Exception\Request("This endpoint may need activating on your account. Please email support@nexmo.com for more information", $status);
```
Comment (afolson) [inline src/Conversion/Client.php]: Do we standardize these exception texts across all of the libraries anywhere? No worries if not, just thinking out loud here.

Comment (mheap) [reply src/Conversion/Client.php]: This is the first support controlled one that I've added. Eventually I'd like to centralise `getException` but the focus is on coverage to start, refactoring second

Comment (arubacao) [reply src/Conversion/Client.php]: @mheap To be honest, I'd suggest a refactoring (or complete rebuild) first. I'm pretty confident that the custom nexmo exceptions are never actually thrown. Guzzle throws a ClientException long before you can even check the http statuscode.

**Pull request merged**